src/components/Module.tsx

```diff
@@ -52,30 +49,26 @@ function Module ({ index, initialData, moduleId, loopId, onChange }: Props) {
     volume: 50,
   }
 
-  const [module, setModule] = useState(initialData || defaultModuleParams)
+  // @ts-ignore Type '{ state: SchedulerContextType; dispatch: Dispatch<any>; }' is not an array type.
+  const [state, dispatch] = useContext(SchedulerContext);
+  const module = state.loop.modules.find((module: any) => module.id === moduleId);
 
-  const update = async (key: string, value: any) => {
```

Comment:
This call is no longer made whenever the user updates.  We have a save button now.